charts/stable/otel-collector/values.yaml

```diff
@@ -0,0 +1,179 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: otel/opentelemetry-collector
+  # -- image tag
+  tag: 0.46.0
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- command is set to `otelcol` when using otel/opentelemetry-collector image but should be
+# changed to `otelcontribcol` if the image is changed to otel/opentelemetry-collector-contrib.
+# @default -- otelcol
+command:
+- "/otelcol"
+
+# -- args is set to the config file which is mounted via the configFileSecret or configFile
+# @default -- Set to the hard-coded volume mount.
+args:
+- "--config=/conf/otel-collector-config.yaml"
+
+# -- environment variables. See more environment variables in the [otel-collector documentation](https://otel-collector.org/docs).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+
+# -- Configures service settings for the chart.
+# @default -- The defaults expose the services needed to receive http and otlp traces
+service:
+  main:
+    enabled: false
+  otlpports:
+    enabled: true
+    type: ClusterIP
+    ports:
+      # Default endpoint for OpenTelemetry gRPC receiver.
+      otlpgrpc:
+        enabled: true
+        protocol: TCP
+        port: 4317
+        targetPort: 4317
+      # Default endpoint for OpenTelemetry HTTP receiver.
+      otlphttp:
+        enabled: true
+        protocol: TCP
+        port: 4318
+        targetPort: 4318
+      # Default endpoint for querying metrics.
+      metrics:
+        enabled: true
+        protocol: TCP
+        port: 8888
+        targetPort: 8888
+
+# -- probes is configured to use an otel extension to get health information from the pod
+# @default -- expects config to include `extensions:health_check:endpoint: 0.0.0.0:13133`
+probes:
+  liveness:
+    enabled: true
+    custom: true
+    spec:
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 1
+      failureThreshold: 3
+      httpGet:
+        path: /
+        port: 13133
+  readiness:
+    enabled: false
+  startup:
+    enabled: false
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # This OTEL Collector is built to trust items within the same cluster so
+  # exposing externally will allow unauthenticated traces to be processed.
+  # @default -- disabled
+  main:
+    enabled: false
+
+# -- Configure persistence settings for the chart under this key.
+# @default -- This application does not require persistence aside from the mounted secret
+persistence: {}
+  # data:
+  #   enabled: false
+  #   mountPath: /data
```

Comment:
If the chart does not require persistence, this key can be removed from the yaml file